releasenotes/notes/support-prompt-annotations-b8e406261197f61a.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    LLM Observability: This introduces the ability to annotate prompt templates via `LLMObs.annotate` on LLM spans.
+    Prompt templates should be passed in as a `Prompt` typed dictionary using `from ddtrace.llmobs.utils import Prompt`
+    or a dictionary following the `Prompt` schema.
```

Comment:
Do we have anything in our corp docs that goes into more detail? I would rather link to this `for more information, see <insert_link_here>...` than potentially dump info on users that might not know what we're talking about here.
```suggestion
    LLM Observability: Introduces the prompt template argument to `LLMObs.annotate()` for LLM spans.
    For more information on prompt template annotations, see <link here>.
```

Reply:
We don't have anything in the corp docs yet. I'm going to a better description of the `Prompt` dict within the `annotate` function's comments, and hopefully that can clarify things until we get some examples up in the corp docs

Reply:
Discussed offline, going to add a link to annotation instructions